run-tests.sh

```diff
@@ -6,3 +6,6 @@ flake8 .
 
 # mypy
 mypy --package hyperframe
+
+# Run tests
+pytest tests --cov
```

Comment:
i think you can add `--cov` to pyproject directly:

```toml
[tool.pytest.ini_options]
addopts = "--cov"
```

Reply:
thats already there actually. removed here: https://github.com/sam-lawrence/HyperFrame/pull/8/commits/e8fdcff74b89277e7e11b2e9c281ad5199f5c0a4

Reply:
removing that then doesn't work. We do have both in beaudaddy though?

Reply:
https://github.com/sam-lawrence/HyperFrame/pull/8/commits/409ee08e2e3e071fda4bf94f81c3a8add1c637f3 reverted

Reply:
might have to add the paths ther eas well https://docs.pytest.org/en/7.1.x/reference/customize.html#pyproject-toml

Reply:
apparently you can do somethign similar with mypy 
```toml
[tool.mypy]
...
packages = ["news_junkie", "stages", "api", "tests"]

[tool.pytest.ini_options]
minversion = "6.0"
addopts = "--cov"
testpaths = ["tests"]
```

Reply:
https://github.com/sam-lawrence/HyperFrame/pull/8/commits/0cb4cfa490d2b13136bc90480a8649c158bb8cc6